src/models/imposter.js

```diff
@@ -54,6 +54,9 @@ function create (Protocol, request) {
 
                 result.requests = server.requests;
                 result.stubs = server.stubs();
+                result.stubs.forEach(function (stub) {
```

Comment:
Got it, although you're already injecting the imposter state in the protocols (e.g. tcpServer, baseHttpServer) by passing in this.state into the stubRepository. This function here, in imposter.js, doesn't inject the imposter state anywhere; it just represents it as JSON coming out of the API, so the developer can see the imposter state. That can be useful for troubleshooting (e.g. to see what the current imposter state is, send a GET to /imposters/:port), but not much else. Since we don't allow initializing the state through the API (appropriately, I think), it doesn't even matter if the user's trying to save off the JSON and re-POST it later.